### DISCUSSION TITLE
Shall GoldWarden support OpenSUSE Tumbleweed?

### DISCUSSION THREAD

**Discussion by beedell-roke** [quexten/goldwarden#7] [Q&A] [answered]
I ask whether it's included as part of "other" per https://github.com/quexten/goldwarden/issues/1#issuecomment-1685369628.

**Comment by quexten** [comment]
I don't have much experience with OpenSuse Tumbleweed, but as far as I know it should be able to install rpms, so #1 will cover it. Until then you can also install it via `go install`.

**Reply by quexten** [reply]
RPMs (and .debs) are now added on each github release.

**Reply by beedell-roke** [reply]
> as far as I know it should be able to install rpms

If those RPMs are tested on something like Fedora instead, compatibility isn't guaranteed.

**Reply by quexten** [reply, marked as answer]
Ah, thanks for pointing that out. I'll test it sometime. Do note that right now the RPM only copies the compiled binary to /usr/bin/ so I'm not sure where it would go wrong. Missing dependencies for fido2/autofill could be a problem. Polkit is not setup (must be setup by goldwarden setup polkit), and running the daemon on user login is not setup (must be setup by goldwarden setup systemd).

**Comment by beedell-roke** [comment]
https://github.com/quexten/goldwarden/issues/12#issue-2031739243
